.travis.yml

```diff
@@ -24,6 +24,9 @@ matrix:
   - if: tag IS present
     env: RUNTYPE=release
     os: linux
+  - if: branch = master
```

Comment:
Adding an extra runtype will make our builds take even longer than they already do. Why not merge this with release?

Reply:
Isn't the release only triggered when we push a new tag ? The doc should be built on all PR or at least every time `/content` is modified

Reply:
Oh sorry, you're right, I thought we did "latest"/"nightly" builds on master.